ostree-upload-server.py

```diff
@@ -171,31 +131,50 @@ def upload(self):
                 if upload.filename == "":
                     return self._response(400, "No filename in request")
 
+                repo_name = request.form.get('repo', None)
+                logging.info("Target repo: {}".format(repo_name))
+
+                if not repo_name:
+                    logging.error("ERROR! Target repo not provided!")
+                    return self._response(400, "ERROR! 'repo' parameter not set!")
+
+                # Verify that this param is not trying anything funky
+                if not UploadWebApp.REPO_NAME_PATTERN.match(repo_name):
+                    logging.error("Target repo name ({}) is not valid!".format(repo_name))
+                    return self._response(400, "ERROR! 'repo' parameter is not valid!")
```

Comment:
This is still checking the repo name against a regex, and I really want this to be explicit.